tests/unit/mainwindow/test_tabwidget.py

```diff
@@ -44,6 +44,7 @@ class TestTabWidget:
             'indicator-width': 3,
             'indicator-padding': configtypes.PaddingValues(2, 2, 0, 4),
             'title-format': '{index}: {title}',
+            'title-alignment': 'left',
```

Comment:
Whoops, my bad - this needs to be the already parsed value, i.e. `Qt.AlignLeft` (you'll also need to add `from PyQt5.QtCore import Qt` to the imports at the top).